iree/compiler/Dialect/Shape/Transforms/test/expand_function_dynamic_dims.mlir

```diff
@@ -36,3 +36,10 @@ func @dynamicReturnInBlock(%arg0 : tensor<2x3xf32>) -> (tensor<?x3xf32>) {
   // CHECK: return %[[RESULT]], %[[SHAPE]] : tensor<?x3xf32>, !shapex.ranked_shape<[?,3]>
   return %1 : tensor<?x3xf32>
 }
+
+// -----
+// CHECK-LABEL: @calls
+func @calls(%arg0 : tensor<1x?x2x?xf32>) -> tensor<1x?x2x?xf32> {
+  %0 = std.call @calls(%arg0) : (tensor<1x?x2x?xf32>) -> tensor<1x?x2x?xf32>
```

Comment:
Can I get a CHECK verifying what this is doing (was looking for the test to verify my assumptions here)?